.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - checkout
       - run:
           name: Install minimal modules globally
-          command: npm i -g lerna gts@1.1.0 typescript
+          command: npm i -g lerna gts@2.0.2 typescript
```

Comment:
I wonder why it was originally running `npm i -g`. This should work right?
```suggestion
          command: npm i
```
Then it will just use the same version as package.json when the next step runs `npm run lint`

Reply:
I believe it was attempting to install the minimum number of required packages to make running CI faster, but yes I can change it.

Reply:
It seems as though it installs from a different package.json, I believe the best option currently might be to just stay with gts 1.1.0.